### DISCUSSION TITLE
Publish statistics on MQTT

### DISCUSSION THREAD

**Discussion by neponn** [mikebrady/shairport-sync#1577] [Ideas]
It would be super cool if the statistics (sync error, net sync, all sync, etc) currently emitted in logs could also be published to the MQTT broker. In my case, I would consume them in Home Assistant to make it easy to monitor SPS performance using charts and subsequently tune buffer parameters.

**Comment by mikebrady** [comment]
Thanks for the suggestion. The most straightforward thing would be to send each as a JSON-type (or is it python?) `{key:value, key:value...}` list. Would that work?

**Reply by neponn** [reply]
JSON would work nicely. HA can consume JSON and generate sensors for each KVP.

**Reply by mikebrady** [reply]
Okay, let me put it “on the list”.
